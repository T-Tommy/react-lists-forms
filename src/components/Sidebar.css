* {
  margin: 0;
  padding: 0;
}

nav {
  background: crimson;
  color: gold;
  font-size: 50px;
  font-weight: bold;
  height: 100vh;
  width: 250px;
  display: flex;
  flex-flow: column wrap;
  justify-content: space-around;
  p {
    text-align: center;
  }
}
