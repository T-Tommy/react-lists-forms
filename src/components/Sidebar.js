import React, { Component } from 'react';
import PropTypes from 'prop-types';
import styles from './Sidebar.css';

export default class Sidebar extends Component {
  static propTypes = {
    children: PropTypes.node
  }

  render() {
    return (
      <nav style={styles}>
        { this.props.children}
      </nav>
    );
  }
}
