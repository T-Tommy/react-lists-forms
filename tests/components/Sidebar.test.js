import React from 'react';
import { shallow } from 'enzyme';
import Sidebar from '../../src/components/Sidebar';

describe('Sidebar component', () => {
  it('renders a Sidebar', () => {
    const wrapper = shallow(
      <Sidebar>
        <p>Home</p>
        <p>About</p>
        <p>Blog</p>
        <p>Contact</p>
      </Sidebar>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
