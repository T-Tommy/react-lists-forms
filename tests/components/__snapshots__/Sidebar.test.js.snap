// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar component renders a Sidebar 1`] = `
<nav
  style={null}
>
  <p>
    Home
  </p>
  <p>
    About
  </p>
  <p>
    Blog
  </p>
  <p>
    Contact
  </p>
</nav>
`;
